src/Razor/src/Microsoft.VisualStudio.Mac.RazorAddin/Microsoft.VisualStudio.Mac.RazorAddin.csproj

```diff
@@ -45,6 +45,12 @@
     <Content Include="$(RepositoryRoot)NOTICE.txt">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    
+    <!-- Embedded grammars -->
+    <Content Include="..\Microsoft.VisualStudio.RazorExtension\EmbeddedGrammars\*">
```

Comment:
Does this include a TMTheme? One thing to be aware of is that the precise set of classification types present on Mac and their colors may be different (though I'd expect any you provide to be the same, unless you omitted them).

Reply:
Ah yes it does, they're directly from the VSWin VSIX. Looks like:
![image](https://user-images.githubusercontent.com/2008729/181360924-f1bc86dc-fe07-4001-8e7e-007216d34ca3.png)

And here's the zipped grammars package if you're curious:
[Grammars.zip](https://github.com/dotnet/razor-tooling/files/9203186/Grammars.zip)

Reply:
oooo we're missing the Razor grammar itself!!!!!

Reply:
The main thing to check is that the classifications that you map to actually exist on Mac. e.g.: if you are theming your HTML grammar using classifications from VS Windows HTML classifications that don't exist on Mac, that code won't get colorized properly.

Reply:
We're definitely relying on classifications from VSWin in that case; however, I'm hoping @ToddGrun and @jimmylewis are keeping those consistent 😄